pkg/kubelet/cm/devicemanager/plugin/v1beta1/server.go

```diff
@@ -141,23 +141,24 @@ func (s *server) SocketPath() string {
 }
 
 func (s *server) Register(ctx context.Context, r *api.RegisterRequest) (*api.Empty, error) {
-	klog.InfoS("Got registration request from device plugin with resource", "resourceName", r.ResourceName)
+	logger := klog.FromContext(ctx)
+	logger.Info("Got registration request from device plugin with resource", "resourceName", r.ResourceName)
 	metrics.DevicePluginRegistrationCount.WithLabelValues(r.ResourceName).Inc()
 
 	if !s.isVersionCompatibleWithPlugin(r.Version) {
 		err := fmt.Errorf(errUnsupportedVersion, r.Version, api.SupportedVersions)
-		klog.InfoS("Bad registration request from device plugin with resource", "resourceName", r.ResourceName, "err", err)
+		logger.Info("Bad registration request from device plugin with resource", "resourceName", r.ResourceName, "err", err)
```

Comment:
should we use logger.Error instead of Info?

Reply:
If it is normal that some error occurs and not some exceptional event that requires administrator attention, then Info is more appropriate.

In general, when converting to structured, contextual logging it is always better to not make semantic changes at the same time. Those are then harder to review and discuss. This here was InfoS before, so sticking with that prior decision is preferred.

Reply:
ok, that make senses.